common/cards/hermits/smajor1995.ts

```diff
@@ -0,0 +1,87 @@
+import {HERMIT_CARDS} from '..'
+import {CardPosModel} from '../../models/card-pos-model'
+import {GameModel} from '../../models/game-model'
+import {RowPos} from '../../types/cards'
+import {RowStateWithHermit} from '../../types/game-state'
+import {getNonEmptyRows} from '../../utils/board'
+import HermitCard from '../base/hermit-card'
+import { applyAilment, removeAilment } from '../../utils/board'
+
+class Smajor1995RareHermitCard extends HermitCard {
+	constructor() {
+		super({
+			id: 'smajor1995_rare',
+			numericId: 161,
+			name: 'Scott',
+			rarity: 'rare',
+			hermitType: 'builder',
+			health: 270,
+			primary: {
+				name: 'Color Splash',
+				cost: ['any'],
+				damage: 30,
+				power: null,
+			},
+			secondary: {
+				name: 'To Dye For',
+				cost: ['any', 'any', 'any'],
+				damage: 70,
+				power: 'Select one of your Hermits. Items attached to this Hermit become any type.',
```

Comment:
rather than having them be any type, which is the same thing command block does, I suggest they all become the type of the hermit card. Also needs to say afk Hermits.

Also, if this is to be done after your attack, it needs to say that